core/sodasql/cli/ingest.py

```diff
@@ -232,15 +229,12 @@ def ingest(
     if not soda_server_client.api_key_id or not soda_server_client.api_key_secret:
         raise ValueError("Missing Soda cloud api key id and/or secret.")
 
-    if tool == 'dbt':
-        dbt_manifest, dbt_run_results = resolve_artifacts_paths(
-            dbt_artifacts=dbt_artifacts,
-            dbt_manifest=dbt_manifest,
-            dbt_run_results=dbt_run_results
-        )
-        test_results_iterator = map_dbt_test_results_iterator(dbt_manifest, dbt_run_results)
-    else:
-        raise NotImplementedError(f"Unknown tool: {tool}")
+    dbt_manifest, dbt_run_results = resolve_artifacts_paths(
```

Comment:
I like that refactoring to a subcommand made this a lot less full of if/elses! 👌